scripts/examples/amm/amm.sh

```diff
@@ -2,15 +2,15 @@
 
 TREASURY=$(elysd keys show -a treasury --keyring-backend=test)
 
-elysd tx amm create-pool elysd tx amm create-pool 10uatom,10uusdt 10000uatom,10000uusdt --swap-fee=0.00 --exit-fee=0.00 --use-oracle=false --from=treasury --keyring-backend=test --chain-id=elystestnet-1 --yes --gas=1000000
+elysd tx amm create-pool elysd tx amm create-pool 10uatom,10uusdt 10000uatom,10000uusdt --swap-fee=0.00 --exit-fee=0.00 --use-oracle=false --from=treasury --keyring-backend=test --chain-id=elysicstestnet-1 --yes --gas=1000000
```

Comment:
these scripts are for local testing. Lets have chain id name different name than testnet or mainnet to avoid any confusion. Same all for below

Reply:
chain-id removed as it can be set in many ways